pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -24,12 +24,16 @@ import (
 
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
+	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
 	internalcache "k8s.io/kubernetes/pkg/scheduler/internal/cache"
 )
 
 // Code is the Status code/type which is returned from plugins.
 type Code int
 
+// PluginToHostPriorityMap declares a map from plugin name to its HostPriorityList.
+type PluginToHostPriorityMap map[string]schedulerapi.HostPriorityList
```

Comment:
I know that HostPriorityList already exists and we may want to use that, but that's not a great name. Let's create a similar struct in the framework API called `NodeScoreList` and rename `PluginToHostPriorityMap` to `PluginToNodeScoreMap`.

Reply:
Done.